client/snap_op.go

```diff
@@ -94,7 +94,12 @@ func (client *Client) Revert(name string, options *SnapOptions) (changeID string
 	return client.doSnapAction("revert", name, options)
 }
 
+var ErrNoDangerous = fmt.Errorf("dangerous option only valid when installing from a file")
```

Comment:
"dangerous option only meaningful when installing from a local file"

ErrNoDangerous isn't super clear to me, ErrInapplicableDangerous?